File: sdks/typescript/src/opik/rest_api/api/types/AlertTriggerConfigWriteType.ts
// 此文件由 Fern 根据我们的 API 定义自动生成。

export const AlertTriggerConfigWriteType = {
    ScopeProject: "scope:project",
    ThresholdFeedbackScore: "threshold:feedback_score",
    ThresholdCost: "threshold:cost",
    ThresholdLatency: "threshold:latency",
    ThresholdErrors: "threshold:errors",
    FilterGuardrailType: "filter:guardrail_type",
} as const;
export type AlertTriggerConfigWriteType =
    (typeof AlertTriggerConfigWriteType)[keyof typeof AlertTriggerConfigWriteType];

File: sdks/python/src/opik/rest_api/types/alert_trigger_config_write_type.py
# 此文件由 Fern 根据我们的 API 定义自动生成。

import typing

AlertTriggerConfigWriteType = typing.Union[
    typing.Literal[
        "scope:project",
        "threshold:feedback_score",
        "threshold:cost",
        "threshold:latency",
        "threshold:errors",
        "filter:guardrail_type",
    ],
    typing.Any,
]
